scss/objects/_grid.scss

```diff
@@ -56,248 +58,232 @@ $DBbreakpoint-small: 480px !default;
+    > %one-quarter    { width: percentage(1 / 4); }
+    > %two-quarters   { width: percentage(2 / 4); }
+    > %three-quarters { width: percentage(3 / 4); }
+
+    > %push-span-all       { margin-left: percentage(1 / 1); }
+    > %push-one-half       { margin-left: percentage(1 / 2); }
+    > %push-one-third      { margin-left: percentage(1 / 3); }
+    > %push-two-thirds     { margin-left: percentage(2 / 3); }
+    > %push-one-quarter    { margin-left: percentage(1 / 4); }
+    > %push-two-quarters   { margin-left: percentage(2 / 4); }
+    > %push-three-quarters { margin-left: percentage(3 / 4); }
+
+    > %pull-span-all       { margin-left: -(percentage(1 / 1)); }
+    > %pull-one-half       { margin-left: -(percentage(1 / 2)); }
+    > %pull-one-third      { margin-left: -(percentage(1 / 3)); }
+    > %pull-two-thirds     { margin-left: -(percentage(2 / 3)); }
+    > %pull-one-quarter    { margin-left: -(percentage(1 / 4)); }
+    > %pull-two-quarters   { margin-left: -(percentage(2 / 4)); }
+    > %pull-three-quarters { margin-left: -(percentage(3 / 4)); }
+
+    // For each of our column groups...
+    @each $group in $DBcol-groups {
+
+        // For each column width from 1 to the column group...
+        @for $i from 1 through $group {
+            $relation: ($i / $group);
+
+            > .#{$DBgrid-column-namespace}--#{$i}-of-#{$group} {
+                @if $relation == 1 {
+                    @extend %span-all;
+                } @else if $relation == 1 {
+                    @extend %one-half;
+                } @else if $relation == (1 / 3) {
+                    @extend %one-third;
+                } @else if $relation == (2 / 3) {
+                    @extend %two-thirds;
+                } @else if $relation == (1 / 4) {
+                    @extend %one-quarter;
+                } @else if $relation == (2 / 4) {
+                    @extend %two-quarters;
+                } @else if $relation == (3 / 4) {
+                    @extend %three-quarters;
+                } @else {
+                    width: percentage($i/$group);
+                }
+            }
+
+            > .#{$DBgrid-column-namespace}--push-#{$i}-of-#{$group} {
+
+                @if $relation == 1 {
+                    @extend %push-span-all;
+                } @else if $relation == (1 / 2) {
+                    @extend %push-one-half;
+                } @else if $relation == (1 / 3) {
+                    @extend %push-one-third;
+                } @else if $relation == (2 / 3) {
+                    @extend %push-two-thirds;
+                } @else if $relation == (1 / 4) {
+                    @extend %push-one-quarter;
+                } @else if $relation == (2 / 4) {
+                    @extend %push-two-quarters;
+                } @else if $relation == (3 / 4) {
+                    @extend %push-three-quarters;
+                } @else {
+                    margin-left: percentage($i/$group);
```

Comment:
`$i/$group` should be written with a single space on each side of the operator: `$i / $group`